imgui-glfw/src/commonMain/kotlin/com/imgui/impl/Utils.kt

```diff
@@ -0,0 +1,45 @@
+package com.imgui.impl
+
+import com.imgui.*
+import com.kgl.glfw.*
+
+expect val isMacOS: Boolean
+expect val isWin32: Boolean
+
+expect fun setupClipboard(ioObj: ImGuiIO, window: Window)
+
+expect fun freeClipboard(ioObj: ImGuiIO)
+
+//FIXME commonize
+internal expect fun initPlatformInterface()
+
+expect var ImGuiViewport.glfwWindow: Window?
+expect var ImGuiViewport.glfwViewportData: ImGuiGlfw.ViewportData?
+
+abstract class ImVector<T> : Iterable<T> {
```

Comment:
Would be neater if this extended `AbstractMutableList` instead of `Iterable`, then we get the iterator and other stuff for free.

Reply:
I explicitly didn't want the `add(index: Int, element: T)` or `removeAt(index: Int)` functions required by `AbstractMutableList` (since those operations are not supported by `ImVector` and we don't need them) I'm only using `Iterable<T>` to enable using forEach

Reply:
`AbstractList`? :thinking:

Reply:
Actually nvm, this can be pimped and codegen'd in another PR.